### ISSUE TITLE
Are there plans to make Oscar 2.1 LTS compatible with Django 3.2 LTS?

### ISSUE THREAD

**Issue by wiesion** [django-oscar/django-oscar#3730]
Comparing the end of extended support dates:
Oscar 2.1 LTS: August 2023 (25 months to go)
Django 3.2 LTS: December 2024 (41 months to go)
Django 3.0 (max. compatible version with Oscar 2.1 LTS): April 2021 (2 months ago)

I wonder if you plan on providing compatibility with Django 3.2 LTS for your 2.1 LTS release?

Sources:
https://github.com/django-oscar/django-oscar#supported-versions
https://www.djangoproject.com/download#supported-versions

**Comment by solarissmoke** [comment]
Hi @wiesion, no objection in principle to making 2.1 compatible with 3.2 - I think it just requires someone who is able to spend a bit of time on that. If you or anyone else is willing/able to make a PR for compatibility I should be able to find time to review/merge and cut a release.

**Comment by wiesion** [comment]
Thank you for the headsup - I did not find the time until now, am at it now.
I have set up the local environment and run tox in parallel: 

```bash
$ git clone {django-oscar}
$ cd {django-oscar}
$ pyenv local 3.5.10 3.6.14 3.7.11 3.8.11 3.9.6
# (Give test settings access to my local Postgres 13 instance)
$ git checkout 2.1.1
$ tox --parallel
```

But i get the following result:
```bash
ERROR:   py35-django22: parallel child exit code 1
ERROR:   py35-django30: parallel child exit code 1
ERROR:   py36-django22: parallel child exit code 1
ERROR:   py36-django30: parallel child exit code 1
  py37-django22: commands succeeded
ERROR:   py37-django30: parallel child exit code 1
  py38-django22: commands succeeded
  py38-django30: commands succeeded
  lint: commands succeeded
  sandbox: commands succeeded
ERROR:   docs: parallel child exit code 1
```

Do i have to do a different setup, or are these errors known to you as well? (For instance, in py36-django30 there are 5 errors, most related to unicode issues.)

**Comment by solarissmoke** [comment]
@wiesion hard to comment on this without knowing specifically what failures you're getting. The tests run fine for me with Django 3.0 on Python 3.6 and 3.7.

**Comment by solarissmoke** [comment]
I've just released version 2.2 on PyPi. Went with 2.2 instead of 2.1.x because there is one backward-incompatible change that projects should be aware of before upgrading, which is mentioned in the release notes.
